src/components/Dashboard/TodoList.vue

```diff
@@ -1027,9 +1020,9 @@ export default class TodoList extends Vue {
         correctedFilingId: correction.correctedFilingId,
         // this is only used for external corrections (IA):
         // Corrections on Alterations always have a correctedFilingType of Alteration -> alterationRequired = true
-        correctedFilingType: EnumUtilities.filingTypeToName(correction.correctedFilingType as FilingTypes, null, true),
+        correctedFilingType: EnumUtilities.filingTypeToName(correction.correctedFilingType as FilingTypes, null, null),
```

Comment:
Eslint was complaining about this since last argument was supposed to be of type `FilingSubTypes`.

Reply:
As the second and third parameters of this method have defaults, you could just omit them here (and below).

Reply:
Fixed, thanks.